### DISCUSSION TITLE
Xiaomi band 7

### DISCUSSION THREAD

**Discussion by Den-Co** [NightscoutFoundation/xDrip#2195] [Q&A]
Please tell me, will support for xiaomi band 7 be added in the next versions?

**Comment by Navid200** [comment]
There is no support for even version 5 or 6.

The plan is for this to be merged:  https://github.com/NightscoutFoundation/xDrip/pull/2004
Then, you can ask the developer of the corresponding app to add support for version 7 to the app.

**Reply by AliQ101** [reply]
Initial version of xDrip integration app/modified xDrip for testing is already out on Artem's blog:
https://bigdigital.home.blog/2022/06/16/watchdrip-a-new-application-for-xdrip-watch-integration/

The updated versions (covering some bugs) are available on Patreon.
